rss-reader/Model/Entity/Channel/FeedChannel.m

```diff
@@ -0,0 +1,62 @@
+//
+//  FeedChannel.m
+//  rss-reader
+//
+//  Created by Uladzislau on 11/18/20.
+//
+
+#import "FeedChannel.h"
+
+NSString *const kRSSChannel = @"channel";
+NSString *const kRSSChannelLink = @"link";
+NSString *const kRSSChannelTitle = @"title";
+NSString *const kRSSChannelDescription = @"description";
+NSString *const kRSSChannelItems = @"RSSChannelItems";
+
+@interface FeedChannel ()
+
+@property (nonatomic, copy, readwrite) NSString *link;
+@property (nonatomic, copy, readwrite) NSString *title;
+@property (nonatomic, copy, readwrite) NSString *summary;
+@property (nonatomic, retain, readwrite) NSArray<FeedItem *> *items;
+
+@end
+
+@implementation FeedChannel
+
++ (instancetype)objectWithDictionary:(NSDictionary *)dictionary {
+    if(!dictionary) {
+        @throw NSInvalidArgumentException;
+        return nil;
+    }
+    
+    FeedChannel *object = [FeedChannel new];
+        
+    object.link = dictionary[kRSSChannelLink];
+    object.title = dictionary[kRSSChannelTitle];
+    object.summary = dictionary[kRSSChannelDescription];
+    object.items = [NSArray arrayWithArray:[dictionary mutableArrayValueForKey:kRSSChannelItems]];
```

Comment:
What is difference with `dictionary[kRSSChannelItems]`?

Reply:
`dictionary[kRSSChannelItems]` doesn't return array explicitly